stdlib/2.7/calendar.pyi

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Iterable, Optional, Tuple
+import datetime
+
+YearType = int
+MonthType = int
+DayType = int
+WeekdayType = int
```

Comment:
_De gustibus non est disputandum_... I found them useful, but I am happy to remove them to adhere to the project's goals.